modules/olmv1-red-hat-catalogs.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_content-type: REFERENCE
+
+[id="olmv1-red-hat-catalogs_{context}"]
+= Red Hat-provided Operator catalogs in OLM 1.0
+
+Operator Lifecycle Manager (OLM) 1.0 does not include Red Hat-provided Operator catalogs by default. If you want to add a Red Hat-provided catalog to your cluster, create a custom resource (CR) for the catalog and apply it to the cluster. The following custom resource (CR) examples show how to create a catalog resources for OLM 1.0.
+
+.Example Red Hat Operators catalog
+[source,yaml,subs="attributes+"]
+----
+apiVersion: catalogd.operatorframework.io/v1alpha1
+kind: Catalog
+metadata:
+  name: redhat-operators
+spec:
+  source:
+    type: image
+    image:
+      ref: registry.redhat.io/redhat/redhat-operator-index:v{product-version}
+----
+
+.Example Certified Operators catalog
+[source,yaml,subs="attributes+"]
+----
+apiVersion: catalogd.operatorframework.io/v1alpha1
+kind: Catalog
+metadata:
+  name: certified-operators
+spec:
+  source:
+    type: image
+    image:
+      ref: registry.redhat.io/redhat/certified-operator-index:v{product-version}
+----
```

Comment:
@jianzhangbjz PTAL at @kevinrizza's comment here: https://github.com/openshift/openshift-docs/pull/66036#discussion_r1362706304

Engineering's preference is to leave off the Marketplace catalog, since it is built by IBM and we have less control if something doesn't work with OLM 1.0 at this stage of development.

If you feel strongly that this should be included, I am happy to open another PR and we can debate that issue there.

Reply:
I created issue OSDOCS-8299. After this PR merges, I will create a follow up PR with the Marketplace catalog, so that we can get the change published before Docs Merge Freeze on 10/27.

Reply:
Follow up Marketplace catalog PR: https://github.com/openshift/openshift-docs/pull/66647

Reply:
ok, thanks!